lib/react.dart

```diff
@@ -1186,6 +1194,12 @@ mixin TypedSnapshot<TSnapshot> {
 
 /// Creates a ReactJS virtual DOM instance (`ReactElement` on the client).
 abstract class ReactComponentFactoryProxy implements Function {
+  /// The JS component factory used by this factory to build [ReactElement]s.
+  ///
+  /// Deprecated: [ReactComponentFactoryProxy]s now use inline [React.createElement()].
```

Comment:
This deprecation message doesn't really spell out what should be used instead.  Can we add an example of what consumers should do if they are using this currently?